include/mgos_event.h

```diff
@@ -101,6 +101,7 @@ enum mgos_event_sys {
    */
   MGOS_EVENT_CLOUD_CONNECTED,
   MGOS_EVENT_CLOUD_DISCONNECTED,
+  MGOS_EVENT_CLOUD_CONNECTING,
```

Comment:
not all cloud connections are MQTT, add a note that it's only supported for MQTT for now.

Reply:
Hi, I've added a note to the code and updated in the other PR. https://github.com/cesanta/mongoose-os/pull/561